app/javascript/controllers/flatpickr_controller.js

```diff
@@ -10,7 +10,7 @@ export default class extends Controller {
       minDate: "today"
     })
     flatpickr(this.endTimeTarget, {
-      minDate: "today"
+      minDate: "tomorrow"
```

Comment:
Great job Margarita! Here, I'm wondering one should be able to book for the same day i.e I can book in the morning in order to spend that night in a flat.

Reply:
Oops, sorry about that, I was playing with various things on the page. Because I wanted to see how we could caclulate the days of the stay and display the total $$$, but didn't get too far.
I agree with you!

Reply:
Ok! Calculating the number of days of stay and displaying the total amount sounds good, we could look into it if there is time.